Dockerfile

```diff
@@ -23,7 +30,7 @@ RUN mix do certs.dev, distillery.release
 RUN tar -czf mongoose_push.tar.gz -C _build/prod/rel/mongoose_push .
 
 
-FROM debian:stretch-slim
+FROM ubuntu:20.04
```

Comment:
This is the final docker image inside which the built MongoosePush will run. We used `debian:stretch-slim` to make the final image as small as possible. Do we have something similar to `ubuntu:20.04` ?

Reply:
Unfortunately, not. I changed the final image back to `debian`.